INFO  21-09-29 06:21:36[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [cluster:76] - Exception in monitor thread while connecting to server 192.168.10.41:27017
com.mongodb.MongoSocketReadException: Prematurely reached end of stream
	at com.mongodb.internal.connection.SocketStream.read(SocketStream.java:112)
	at com.mongodb.internal.connection.SocketStream.read(SocketStream.java:131)
	at com.mongodb.internal.connection.InternalStreamConnection.receiveResponseBuffers(InternalStreamConnection.java:647)
	at com.mongodb.internal.connection.InternalStreamConnection.receiveMessageWithAdditionalTimeout(InternalStreamConnection.java:512)
	at com.mongodb.internal.connection.InternalStreamConnection.receiveCommandMessageResponse(InternalStreamConnection.java:355)
	at com.mongodb.internal.connection.InternalStreamConnection.receive(InternalStreamConnection.java:304)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.lookupServerDescription(DefaultServerMonitor.java:218)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:144)
	at java.lang.Thread.run(Unknown Source)
INFO  21-09-29 06:21:38[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [cluster:76] - Exception in monitor thread while connecting to server 192.168.10.41:27017
com.mongodb.MongoSocketOpenException: Exception opening socket
	at com.mongodb.internal.connection.SocketStream.open(SocketStream.java:70)
	at com.mongodb.internal.connection.InternalStreamConnection.open(InternalStreamConnection.java:143)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.lookupServerDescription(DefaultServerMonitor.java:188)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:144)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at com.mongodb.internal.connection.SocketStreamHelper.initialize(SocketStreamHelper.java:107)
	at com.mongodb.internal.connection.SocketStream.initializeSocket(SocketStream.java:79)
	at com.mongodb.internal.connection.SocketStream.open(SocketStream.java:65)
	... 4 common frames omitted
INFO  21-09-29 06:22:00[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [connection:71] - Opened connection [connectionId{localValue:15, serverValue:1}] to 192.168.10.41:27017
INFO  21-09-29 06:22:00[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [cluster:71] - Monitor thread successfully connected to server with description ServerDescription{address=192.168.10.41:27017, type=STANDALONE, state=CONNECTED, ok=true, minWireVersion=0, maxWireVersion=8, maxDocumentSize=16777216, logicalSessionTimeoutMinutes=30, roundTripTimeNanos=931500}
INFO  21-09-29 06:22:08[cluster-rtt-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [connection:71] - Opened connection [connectionId{localValue:16, serverValue:7}] to 192.168.10.41:27017
INFO  21-09-29 09:31:10[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [cluster:76] - Exception in monitor thread while connecting to server 192.168.10.41:27017
com.mongodb.MongoNodeIsRecoveringException: Command failed with error 11600 (InterruptedAtShutdown): 'interrupted at shutdown' on server 192.168.10.41:27017. The full response is {"ok": 0.0, "errmsg": "interrupted at shutdown", "code": 11600, "codeName": "InterruptedAtShutdown"}
	at com.mongodb.internal.connection.ProtocolHelper.createSpecialException(ProtocolHelper.java:242)
	at com.mongodb.internal.connection.ProtocolHelper.getCommandFailureException(ProtocolHelper.java:171)
	at com.mongodb.internal.connection.InternalStreamConnection.receiveCommandMessageResponse(InternalStreamConnection.java:358)
	at com.mongodb.internal.connection.InternalStreamConnection.receive(InternalStreamConnection.java:304)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.lookupServerDescription(DefaultServerMonitor.java:218)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:144)
	at java.lang.Thread.run(Unknown Source)
INFO  21-09-29 09:31:22[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [cluster:76] - Exception in monitor thread while connecting to server 192.168.10.41:27017
com.mongodb.MongoSocketOpenException: Exception opening socket
	at com.mongodb.internal.connection.SocketStream.open(SocketStream.java:70)
	at com.mongodb.internal.connection.InternalStreamConnection.open(InternalStreamConnection.java:143)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.lookupServerDescription(DefaultServerMonitor.java:188)
	at com.mongodb.internal.connection.DefaultServerMonitor$ServerMonitorRunnable.run(DefaultServerMonitor.java:144)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at com.mongodb.internal.connection.SocketStreamHelper.initialize(SocketStreamHelper.java:107)
	at com.mongodb.internal.connection.SocketStream.initializeSocket(SocketStream.java:79)
	at com.mongodb.internal.connection.SocketStream.open(SocketStream.java:65)
	... 4 common frames omitted
INFO  21-09-29 09:31:45[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [connection:71] - Opened connection [connectionId{localValue:21, serverValue:7}] to 192.168.10.41:27017
INFO  21-09-29 09:31:45[cluster-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [cluster:71] - Monitor thread successfully connected to server with description ServerDescription{address=192.168.10.41:27017, type=STANDALONE, state=CONNECTED, ok=true, minWireVersion=0, maxWireVersion=8, maxDocumentSize=16777216, logicalSessionTimeoutMinutes=30, roundTripTimeNanos=1102200}
INFO  21-09-29 09:31:52[cluster-rtt-ClusterId{value='6152d8deb2ca8b4f02889463', description='null'}-192.168.10.41:27017] [connection:71] - Opened connection [connectionId{localValue:22, serverValue:9}] to 192.168.10.41:27017
INFO  21-09-29 14:46:56[http-nio-9111-exec-9] [Cookie:173] - A cookie header was received [37.4375268; _gid=GA1.1.1839116724.1632881326] that contained an invalid cookie. That cookie will be ignored.
 Note: further occurrences of this error will be logged at DEBUG level.
